Makefile

```diff
@@ -103,19 +103,17 @@ ready: fmt-amend ## Amend current commit with fmt changes
 
 ##@ Documentation
 
-ANTORA_DOCS=./antora/docs
-ANNOTATIONS_JSON=$(ANTORA_DOCS)/rule_annotations.json
-
-$(ANNOTATIONS_JSON):
-# Use jq sort_by here because otherwise the order is different every time
-	@opa inspect --annotations --format json $(POLICY_DIR) | jq -S '.annotations |= sort_by(.location.file, .location.row)' > $@
-
-.PHONY: clean-annotations
-clean-annotations:
-	@rm -f $(ANNOTATIONS_JSON)
+# These two should produce identical output
+# (For debugging only; we don't need these for anything important.)
+#
+.PHONY: annotations-js
+# opa-inspect-js produces some duplicate records hence the unique_by
+annotations-js:
+	@node antora/hack/inspect-annotations.js | jq 'unique_by(.) | sort_by(.location.file, .location.row)'
```

Comment:
FYI @zregvart re the `unique_by` here. There are dupes coming from the module, I think - maybe due to each rule including the package scoped annotations in its chain.